trade-finance-manager-api/src/v1/api.js

```diff
@@ -622,6 +621,21 @@ const addUnderwriterCommentToGefDeal = async (dealId, commentType, comment) => {
   }
 };
 
+const getAllFacilities = async () => {
+  try {
+    const response = await axios({
+      method: 'GET',
+      url: `${centralApiUrl}/v1/tfm/facilities`,
+      headers: {
+        'Content-Type': 'application/json',
+      },
+    });
+    return response.data;
+  } catch ({ response }) {
+    return response;
```

Comment:
Please add `console.error('Error in getting all facilities', { response });`

Reply:
fixed